charts/stable/vikunja/values.yaml

```diff
@@ -160,21 +127,11 @@ service:
   main:
     ports:
       main:
-        port: 10220
+        port: 3456
 persistence:
   files:
     enabled: true
     mountPath: /app/vikunja/files
-  vikunja-nginx:
-    enabled: true
-    noMount: true
-    type: configmap
-    objectName: nginx-config
-    targetSelector:
-      main:
-        proxy:
-          mountPath: /etc/nginx/conf.d/default.conf
-          subPath: nginx-config
```

Comment:
This configmap also needs to be removed as object.
Not only the mount

Reply:
Thanks i just done that